lib/namespace.ts

```diff
@@ -252,7 +255,10 @@ export class Namespace<
    * @return a new {@link BroadcastOperator} instance for chaining
    */
   public to(room: Room | Room[]) {
-    return new BroadcastOperator<EmitEvents, SocketData>(this.adapter).to(room);
+    return new BroadcastOperator<
+      DecorateAcknowledgementsWithMultipleResponses<EmitEvents>,
```

Comment:
Couldn't we move the `DecorateAcknowledgementsWithMultipleResponses` type to the `BroadcastOperator` class, so we don't have to apply it everytime?

Reply:
I definitely think that would be nicer; however after looking at the code, I don't think we can.
The `BroadcastOperator` has an `expectSingleResponse` flag, which changes the behavior between single responses and multiple. With that, the options I see are to either: apply the decorations when switching to a `BroadcastOperator`, or to setup the expectSingleResponse as a third generic type parameter on the BroadcastOperator.